modules/core/src/null-loader.ts

```diff
@@ -21,6 +21,11 @@ export const NullWorkerLoader: Loader = {
   }
 };
 
+function passthrough(arrayBuffer, options, context) {
```

Comment:
JSDoc comment. Explain why this needs to be duplicated here in addition to parseWithWorker.